### ISSUE TITLE
stroke size controlling

### ISSUE THREAD

**Issue by godofdream** [czczup/URST#5]
In your paper you mention changing the decoder of adain to change the stroke size.
what is the difference between the decoders e.g. "decoder_stroke_perceptual_loss_1.pth" ?
In my case, I would like to convert an ultra high resolution (8192x8192), but applying the same "huge" stroke size, as I would if I would resize the picture to 1024x1024

**Comment by czczup** [comment]
Hi, using the model trained with stroke perceptual loss can increase the stroke size to a certain extent.
But due to the limitation of the effective receptive field, 8192x8192 images still can't get the stroke size as 1024x1024 image.

**Comment by godofdream** [comment]
I understand. Would this be overcomeable with TINs of the TIN? so basically creating a pyramid?
Also, do you think https://github.com/LouieYang/stroke-controllable-fast-style-transfer could be adapted for URST?
Did you think about spatial control in URST?

(By the way, URST is impressive, Congrats)

**Comment by czczup** [comment]
Yes, URST can be adapted to stroke-controllable-fast-style-transfer. The only problem is that this code is TensorFlow, I don't know how to implement it immediately.
I have tried to adapt URST to another stroke control method - [Multimodal Transfer: A Hierarchical Deep Convolutional Neural Network for Fast Artistic Style Transfer](https://github.com/FeliMe/multimodal_style_transfer). It's very easy to implement, and we can get very large brush strokes. I would clean up the code and make it public in recent days.

**Comment by RahulBhalley** [comment]
Hi @czczup thanks for your work! You mentioned the following, so have you made any progress on it?

> I have tried to adapt URST to another stroke control method - [Multimodal Transfer: A Hierarchical Deep Convolutional Neural Network for Fast Artistic Style Transfer](https://github.com/FeliMe/multimodal_style_transfer). It's very easy to implement, and we can get very large brush strokes. I would clean up the code and make it public in recent days.

**Comment by czczup** [comment]
> Hi @czczup thanks for your work! You mentioned the following, so have you made any progress on it?
> 
> > I have tried to adapt URST to another stroke control method - [Multimodal Transfer: A Hierarchical Deep Convolutional Neural Network for Fast Artistic Style Transfer](https://github.com/FeliMe/multimodal_style_transfer). It's very easy to implement, and we can get very large brush strokes. I would clean up the code and make it public in recent days.

Hello, thanks for your attention. I'm sorry I can't find the code implemented at that time. I can try to re-implement it, but it will take some time.

**Comment by RahulBhalley** [comment]
It's okay. Thanks for reply.

**Comment by czczup** [comment]
> It's okay. Thanks for reply.

Hi, [Multimodal Transfer](https://github.com/FeliMe/multimodal_style_transfer) is ready now.
https://github.com/czczup/URST/tree/main/Wang2017Multimodal

**Comment by RahulBhalley** [comment]
> > It's okay. Thanks for reply.
> 
> 
> 
> Hi, [Multimodal Transfer](https://github.com/FeliMe/multimodal_style_transfer) is ready now.
> 
> https://github.com/czczup/URST/tree/main/Wang2017Multimodal

I'm gonna go back read that paper. Got distracted to stable diffusion buzz. Thanks for your hard work!!
